internal/pkg/auth/user_login.go

```diff
@@ -168,9 +177,12 @@ func AuthorizeUser(p *print.Printer, isReauthentication bool) error {
 	if err != nil {
 		return fmt.Errorf("open browser to URL %s: %w", authorizationURL, err)
 	}
+	p.Debug(print.DebugLevel, "opening browser for authentication")
```

Comment:
these two logs should go above the call to `openBrowser`, otherwise if there is an error we don't see them